apps/modernization-ui/src/apps/search/investigation/InvestigationSearch.tsx

```diff
@@ -1,35 +1,58 @@
-import { useEffect } from 'react';
-import { useForm } from 'react-hook-form';
-import { Investigation } from 'generated/graphql/schema';
 import { SearchLayout, SearchResultList } from 'apps/search/layout';
-import { InvestigationSearchResultListItem } from './result/list';
+import InvestigationSearchForm from './InvestigationSearchForm';
+import { useForm } from 'react-hook-form';
 import { InvestigationFilterEntry } from './InvestigationFormTypes';
 import { useInvestigationSearch } from './useInvestigationSearch';
+import { useEffect } from 'react';
+import { Investigation } from 'generated/graphql/schema';
+import { InvestigationSearchResultListItem } from './result/list';
+import { InvestigationFormContext } from './InvestigationFormContext';
 
-const InvestigationSearch = () => {
-    const { handleSubmit, reset: resetForm } = useForm<InvestigationFilterEntry, Partial<InvestigationFilterEntry>>({
-        mode: 'onBlur'
-    });
+const defaultSelectable = { name: '', value: '', label: '' };
+const defaultValues: InvestigationFilterEntry = {
+    createdBy: defaultSelectable,
+    updatedBy: defaultSelectable,
+    investigator: defaultSelectable,
+    pregnancyStatus: defaultSelectable,
+    investigationStatus: defaultSelectable,
+    jurisdictions: [defaultSelectable],
+    conditions: [defaultSelectable],
+    caseStatuses: [defaultSelectable],
+    notificationStatuses: [defaultSelectable],
+    outbreaks: [defaultSelectable],
+    processingStatuses: [defaultSelectable],
+    programAreas: [],
+    reportingFacility: defaultSelectable
+};
 
+const InvestigationSearch = () => {
     const { status, search, reset, results } = useInvestigationSearch();
+    const form = useForm<InvestigationFilterEntry, Partial<InvestigationFilterEntry>>({
+        mode: 'all',
+        defaultValues
+    });
 
     useEffect(() => {
         if (status === 'waiting') {
-            resetForm();
+            form.reset();
         }
-    }, [resetForm, status]);
+    }, [form.reset, status]);
 
     return (
         <SearchLayout
-            criteria={() => <div>criteria</div>}
+            criteria={() => (
+                <InvestigationFormContext.Provider value={form}>
+                    <InvestigationSearchForm />
+                </InvestigationFormContext.Provider>
```

Comment:
There is a `FormProvider` and `useFormContext` hook, you can see it in action [here](https://github.com/CDCgov/NEDSS-Modernization/blob/f467a25fd1bb139cf651fa8c393ed9c2793a6ddc/apps/modernization-ui/src/apps/search/patient/PatientSearch.tsx#L31)

Reply:
fixed